src/components/ebay-menu/README.md

```diff
@@ -0,0 +1,52 @@
+# ebay-menu
+
+## ebay-menu Tag
+
+### ebay-menu Usage
+
+```marko
+<ebay-menu text="text">
+    <ebay-menu-item>item 1</ebay-menu-item>
+    <ebay-menu-item>item 2</ebay-menu-item>
+    <ebay-menu-item>item 3</ebay-menu-item>
+</ebay-menu>
+```
+
+### ebay-menu Attributes
+
+Name | Type | Stateful | Description
+--- | --- | --- | ---
+`type` | String | No | Can be "fake" / "radio" / "checkbox"
+`reverse` | Boolean | No | expand menu flyout to the left
+`fix-width` | Boolean | No | constrain items container width to button width
+`size` | String | No | button size, "small" or "large" (default: medium)
```

Comment:
There is no more medium button.  Default is small and they can override with "large"

Reply:
Okay, let me just update the docs. The string is a passthrough for the class sizing.